auth.mkd

```diff
@@ -172,43 +172,43 @@ Valid Not Before (VNB) Timestamp by UA (4-bytes):
+{{ua-attestation}}, {{dime-attestation}}, and {{dime-hierarchy}} complete the trust chain, but the chain cannot yet be trusted as having any relevance to the observed UA because replay attacks are trivial. At this point, the key nominally possessed by the UA is trusted but the UA has not yet been proven to possess that private key.
 
-It is necessary for the UA to prove possession by dynamically signing data that is unique and unpredictable but easily verified by the Observer. This can be in the form of a DRIP Wrapper or Manifest ({{drip-wrapper}}, {{drip-manifest}}) containing an ASTM Message that fulfills the requirements. Verification of this signed data MUST be performed by the Observer as part of the received UAS RID information trust assessment ({{trust-assessment}}).
+It is necessary for the UA to prove possession by dynamically signing data that is unique and unpredictable but easily verified by the Observer. This can be a DRIP Wrapper or Manifest ({{drip-wrapper}}, {{drip-manifest}}) containing an ASTM Message that fulfills the requirements. Verification of this signed data MUST be performed by the Observer as part of the received UAS RID information trust assessment ({{trust-assessment}}).
 
 ## ASTM Authentication Message {#auth-message}
 
-The Authentication Message (Message Type 0x2) is a unique message in the {{F3411}} Broadcast standard as it is the only one that can be larger than the Legacy Transport size. To address this, it is defined as a set of "pages", each of which fits into a single Legacy Transport frame. For Extended Transports these pages are still used but are all in a single frame.
+The Authentication Message (Message Type 0x2) is  unique in the {{F3411}} Broadcast standard as it is the only message that can be larger than the Legacy Transport size. To address this, it is defined as a set of "pages," each of which fits into a single Legacy Transport frame. For Extended Transports these pages are still used but are all in a single frame.
 
 ### Authentication Page {#auth-page}
 
-This document leverages Authentication Type 0x5, Specific Authentication Method (SAM), as the principal authentication container, defining a set of SAM Types in {{drip-authentication-formats}}. This is denoted in every Authentication Page in the `Page Header`. The SAM Type is denoted as a field in the `Authentication Payload` (see {{sam-data}}).
+This document leverages Authentication Type 0x5, Specific Authentication Method (SAM), as the principal authentication container, defining a set of SAM Types in {{drip-authentication-formats}}. This is encoded in every Authentication Page in the `Page Header`. The SAM Type is defined as a field in the `Authentication Payload` (see {{sam-data}}).
```

Comment:
"This is encoded..": Please clarify what "This" refers to.

Reply:
Fixed in 807ce54c638b588c98c18d0dc51e805cb7ae2667